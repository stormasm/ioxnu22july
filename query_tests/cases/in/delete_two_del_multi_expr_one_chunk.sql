-- Demonstrate soft deleted rows will not be return to queries
-- IOX_SETUP: TwoDeletesMultiExprsOneChunk

-- select *
SELECT * from cpu;

SELECT foo from cpu;

SELECT * from cpu where cast(time as bigint) > 30;

SELECT count(bar) from cpu where cast(time as bigint) > 30;

SELECT * from cpu where cast(time as bigint) > 40;

SELECT max(time) from cpu where cast(time as bigint) > 40;
