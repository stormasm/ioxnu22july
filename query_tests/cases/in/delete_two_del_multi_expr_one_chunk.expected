-- Test Setup: TwoDeletesMultiExprsOneChunk
-- SQL: SELECT * from cpu;
+-----+-----+--------------------------------+
| bar | foo | time                           |
+-----+-----+--------------------------------+
| 1   | me  | 1970-01-01T00:00:00.000000040Z |
+-----+-----+--------------------------------+
-- SQL: SELECT foo from cpu;
+-----+
| foo |
+-----+
| me  |
+-----+
-- SQL: SELECT * from cpu where cast(time as bigint) > 30;
+-----+-----+--------------------------------+
| bar | foo | time                           |
+-----+-----+--------------------------------+
| 1   | me  | 1970-01-01T00:00:00.000000040Z |
+-----+-----+--------------------------------+
-- SQL: SELECT count(bar) from cpu where cast(time as bigint) > 30;
+----------------+
| COUNT(cpu.bar) |
+----------------+
| 1              |
+----------------+
-- SQL: SELECT * from cpu where cast(time as bigint) > 40;
++
++
-- SQL: SELECT max(time) from cpu where cast(time as bigint) > 40;
+---------------+
| MAX(cpu.time) |
+---------------+
|               |
+---------------+
